Tests/SwiftFormatPrettyPrintTests/RepeatStmtTests.swift

```diff
@@ -17,12 +17,14 @@ public class RepeatStmtTests: PrettyPrintTestCase {
       repeat {
         let a = 123
         var b = "abc"
-      } while condition
+      }
+      while condition
       repeat {
         let a = 123
         var b = "abc"
-      } while condition &&
-              condition2
+      }
+      while condition &&
+        condition2
 
```

Comment:
Can you add an inline `repeat-while` test case?

Reply:
Done.